docs/javascript/v2/quickstart/react-quickstart.mdx

```diff
@@ -152,6 +154,8 @@ export default function App() {
 }
 ```
 
+For all the CSS styling used in the example, you can refer to the styles.css file in CodeSandbox [here](https://codesandbox.io/s/github/100mslive/100ms-examples/tree/main/web/react-quickstart?file=/src/styles.css) 
```

Comment:
better to add tabs .. and import statement in `App.jsx` rather than asking to go to codesandbox ? 
<img width="743" alt="Screenshot 2023-05-03 at 2 26 29 PM" src="https://user-images.githubusercontent.com/10416686/235872666-6bfcc394-bac0-41f7-960b-39b780c51e8b.png">